manuscript/paper.tex

```diff
@@ -531,20 +530,21 @@ \subsection{Inference of selective sweep dataset (Suppl. Fig.
     = 10,000}. Default values were used for all other inference parameters.
 
 \subsection{Inference of 1000 Genomes dataset (Suppl. Fig.
-    \ref{fig:Supplementary_Figure_2}, Suppl. Fig.
-    \ref{fig:Supplementary_Figure_3})} The 1000 Genomes dataset was downloaded from
-\textcolor{red}{[insert link]}. Ancestral fasta sequence for chromosome 20
-(GRCh38) was downloaded from the Ensembl database. Inference was performed with
-a Snakemake pipeline \textcolor{red}{(cite Ben's repo?)} using \texttt{tsinfer
-    version 0.3.3} for the long arm of chromosome 20 after filtering out duplicate
-variant positions, variants with missing or low quality ancestral allele,
-singletons, n-1-tons and n-2-tons. Only bi-allelic SNPs were included for
-inference. For the inference in Suppl. Fig. 2A, all genomic regions were
-included. For the inference in Suppl. Figs 2B and 3, genomic regions with a
-site density lesser than 5 per kb were discarded. For Suppl. Fig. 2,
-\texttt{tsdate version 0.2.1} was used to estimate the age of ancestral nodes
-with \textit{mutation rate = 1.29e-8}, setting all other parameters to default
-values.
+\ref{fig:Supplementary_Figure_2}, Suppl. Fig.
+\ref{fig:Supplementary_Figure_3})} The 1000 Genomes dataset was downloaded from
+\texttt{\url{https://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20220422_3202_phased_SNV_INDEL_SV/}}.
+Ancestral fasta sequences for chromosomes 17 and 20 (GRCh38) were downloaded
+from the Ensembl database. Inference was performed with a Snakemake pipeline
+(\texttt{\url{https://github.com/benjeffery/tsinfer-snakemake/}}) using
+\texttt{tsinfer version 0.3.3} for the long arms of chromosome 17 and 20 after
+filtering out duplicate variant positions, variants with missing or low quality
+ancestral allele, singletons, n-1-tons and n-2-tons. Only bi-allelic SNPs were
+included for inference. For the inference in Suppl. Fig. 2, all genomic regions
```

Comment:
Better be consistent and use the \refs here rather than hard-coding the references.

Reply:
Ah, I see that's how it was before - well might as well fix it while you're in here!